Client/src/app/components/historyComponent/history.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="history-title">Watch History</div>
+
+<div class="history-list" *ngIf="historyReady">
+  <p *ngFor="let video of httpService.history" (click)="playVideo(video.youTubeId)" class="history-videos">
+  {{video.name}}
+  <button class="button" (click)="deleteHistory(video.youTubeId)">X</button>
+</p>
+  <div *ngIf="httpService.history===[]">
```

Comment:
This ngIf should use a function instead of making the compration directly, both for performance and readability